src/libraries/Common/src/Interop/Unix/System.Native/Interop.GetPwUid.cs

```diff
@@ -20,6 +24,74 @@ internal unsafe struct Passwd
             internal byte* Shell;
         }
 
+        /// <summary>
+        /// Gets the user name associated to the current user's UID.
+        /// </summary>
+        /// <returns>On success, return a string with the user name. On failure, returns an empty string.</returns>
+        internal static unsafe string GetUserNameFromPasswd()
```

Comment:
This was the existing code from Environment.Unix.cs, which is also consumed by tar now.